### PR TITLE
Add all services to the Service namespace

### PR DESCRIPTION
**Pull request by lioncash**
Previously there was a split where some of the services were in the Service namespace and others were not.

### CODE REVIEW

File: src/core/hle/service/ac_u.cpp [hunk 1]
```diff
@@ -2,14 +2,14 @@
 // Licensed under GPLv2 or any later version
 // Refer to the license.txt file included.
 
+#include <array>
```
Comment (InusualZ) [inline src/core/hle/service/ac_u.cpp]: This is used?

Comment (lioncash) [inline src/core/hle/service/ac_u.cpp]: Line 15

Comment (InusualZ) [inline src/core/hle/service/ac_u.cpp]: Ops, my bad.

File: src/core/hle/service/gsp_gpu.cpp [hunk 2]
```diff
@@ -18,13 +18,11 @@
...
-const static u32 REGS_BEGIN = 0x1EB00000;
-
-////////////////////////////////////////////////////////////////////////////////////////////////////
-// Namespace GSP_GPU
+namespace Service {
+namespace GSP {
 
-namespace GSP_GPU {
+// Beginning address of HW regs
+const u32 REGS_BEGIN = 0x1EB00000;
```
Comment (linkmauve) [inline src/core/hle/service/gsp_gpu.cpp]: What is the reason for dropping the static specifier here?  This symbol doesn’t seem to be used anywhere else.

Comment (lioncash) [inline src/core/hle/service/gsp_gpu.cpp]: const internally links by default; static isn't necessary.

File: src/core/hle/service/soc_u.cpp [hunk 3]
```diff
@@ -53,12 +53,10 @@
...
 
-static const s32 SOCKET_ERROR_VALUE = -1;
-
-////////////////////////////////////////////////////////////////////////////////////////////////////
-// Namespace SOC_U
+namespace Service {
+namespace SOC {
 
-namespace SOC_U {
+static const s32 SOCKET_ERROR_VALUE = -1;
```
Comment (linkmauve) [inline src/core/hle/service/soc_u.cpp]: Then this one should drop static, right?

Comment (lioncash) [inline src/core/hle/service/soc_u.cpp]: Done

**Pull request merged**